doc/doc-site/docs/off-chain/data-service-stack.md

```diff
@@ -1,3 +1,73 @@
 # Data service stack
 
-Stand by for more info on the Econia data service stack, which provides REST and WebSocket endpoints for Econia market and user data.
+The data service stack (DSS), is a group of programs that power the Econia off-chain products.
+It exposes a REST API and a WebSocket server, and is composed of multiple services (database, API server, WebSocket server, indexer, …).
+To make it easy to run, we have dockerized each service, and have created a Docker compose file to run everything easily.
+On this page, we're going to see how to set everything up.
+
+# How it works
+
+We have a REST API and a WebSocket server.
+
+The WebSocket server mainly notifies you of any events emitted by the contract.
+
+The REST API also contains all the events emitted by the contract, as well as some processed data, like order history, order book state, …
+
+In order to access the WebSocket server, you have to connect to the following URL: `ws://your-host/[JWT]` where `[JWT]` is a JWT.
+To obtain said JWT, you need to query the REST API (or you can manually generate it, you can find the secret in the Docker compose file).
```

Comment:
How about "To get the JWT you can either query the REST API, or manually generate one yourself by looking up the plaintext secret in the local Docker compose file.